### ISSUE TITLE
more colour blind friendly colour scheme 

### ISSUE THREAD

**Issue by NoSitRecords** [polyend/TrackerBetaTesting#1539]
For version 1.6.0b6
I love the new colour scheme but as I'm colour blind it's hard to see the red mark, when you hit REC it's hard to follow the scroller, if it's possible to add an option to select different colours for the scroller and the background so that people like me can use it more easily.

**Comment by Jacknametrouble** [comment]
Thank you for your feedback.
